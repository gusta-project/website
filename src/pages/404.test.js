import React from 'react';
import renderer from 'react-test-renderer';

import NotFoundPage from './404';

jest.mock('components/Header', () => require('utils').mockComponent('Header'));
jest.mock('components/Footer', () => require('utils').mockComponent('Footer'));

describe('NotFoundPage', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NotFoundPage />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
