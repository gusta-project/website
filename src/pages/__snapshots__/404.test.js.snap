// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
Array [
  <Header />,
  <div
    className="container"
  >
    <main>
      <h1>
        not_found_404
      </h1>
      <p>
        not_found_stmt
      </p>
    </main>
    <Footer />
  </div>,
]
`;
